robyn/templating.py

```diff
@@ -28,5 +29,52 @@ def render_template(self, template_name, **kwargs) -> Response:
             headers=Headers({"Content-Type": "text/html; charset=utf-8"}),
         )
 
+    def add_template_global(self, func: Callable, name: str | None = None):
```

Comment:
```suggestion
    def add_template_global(self, func: Callable, name: Optional[str]=None):
```

`|` syntax is not supported in Python3.8 and we have to maintain backward compatibility from 3.8 onwards.